src/server/constants.py

```diff
@@ -0,0 +1,24 @@
+import os
+
+def defaultEnv(key: str, default: str = ""):
```

Comment:
This public function can be simplified to:
```python
def defaultEnv(key: str, default: str = "") -> str:
    return os.getenv(key, default)
```

To be honest this function is not necessary, since it can be shortened to a one-liner.
If you decide to leave it be (for some reason), please add a docstring (as a linter suggests) and a typehint to the returned value (as shown in the snippet above).